js/index.js

```diff
@@ -1,7 +1,114 @@
+}
+
+//Bonus 1
+var lorem = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vulputate neque enim, quis hendrerit ex ultricies nec. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aliquam ut pretium ex. Sed massa nisl, auctor a est sit amet, egestas rutrum nisl. Nam ac turpis sed lectus auctor malesuada ut ac nisl. Curabitur egestas sagittis quam, non tincidunt ipsum molestie eget. Nullam viverra hendrerit vulputate. Donec congue dui vel mauris pretium accumsan. Sed a metus sit amet magna euismod tempor sit amet vitae urna. Curabitur a velit porttitor, blandit risus vitae, laoreet nisl. Curabitur lacinia malesuada lobortis. Mauris dictum dui at luctus dictum. Nullam interdum, quam sit amet imperdiet gravida, purus odio accumsan mi, vel convallis sapien quam eu dolor. Cras eget felis accumsan, ullamcorper diam ac, eleifend lectus. Donec facilisis magna placerat neque luctus, nec faucibus nulla feugiat. Quisque vehicula, felis sed volutpat fringilla, nulla nibh dapibus felis, id faucibus nulla risus a eros. Interdum et malesuada fames ac ante ipsum primis in faucibus. Ut pretium nibh quis consequat eleifend. Integer ligula tortor, molestie sed risus eget, bibendum rhoncus augue. Mauris eleifend, sapien eget porta tempor, risus sem cursus arcu, eu consequat sapien quam a erat. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aliquam ut tortor vel nisi blandit varius. Quisque ullamcorper feugiat urna, eget pretium augue aliquam in. In sed neque ligula. Morbi tristique, sapien eget sagittis eleifend, urna felis molestie ex, vehicula scelerisque lectus libero vitae sapien. Nulla nec congue augue, convallis tincidunt justo. Praesent sollicitudin sem non vehicula iaculis. Pellentesque sed lacinia purus, vitae ullamcorper risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Cras dolor lacus, finibus a sem a, laoreet gravida ex. Fusce ut molestie enim, in euismod libero. Etiam id fringilla nulla, at venenatis justo. Maecenas at erat euismod, porttitor nisi a, euismod erat. Duis non risus eros. Aenean ultrices felis sit amet urna tincidunt faucibus. Etiam sit amet augue eleifend, porta lacus vitae, commodo eros. Cras tempor auctor dignissim. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean sit amet lorem et velit pharetra varius vitae ut libero. Ut dapibus dolor non sem volutpat, eu lobortis nibh cursus.`;
+
+//this works if I remove the paragraphs, as I am counting white spaces for words. not sure how to catch the 2 words after a paragraph break.
+var wordCount = 0;
+var searchTerm = ` et `;
+var etCount = 0;
+var nextIndexOfTerm = lorem.indexOf(searchTerm); //first occurence of search term. returns -1 if none found.
+
+for (var i = 0; i < lorem.length; i++) {
+  if (lorem[i] === ` `) {
+    wordCount++;
+  }
+}
+wordCount++; //this is for the last word of the string which is not succeeded by a space.
+console.log(`Number of words in string: ${wordCount}`);
```

Comment:
Good solution
Another way would be, instead of checking the blank spaces (which may lead to some error), you could convert your string "lorem" to an array with the method .split(" "), and just print the length of the array.